say.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+var Debug = false // override with --debug parameter
+
+func parseDebug(args []string) {
+	// debug needs to be parsed at the beginning to have DEBUG enabled as quickly as possible
+	// otherwise, parsing others or other parameters don't have debug enabled
+	for i := 0; i < len(args); i++ {
+		if args[i] == "--debug" {
+			Debug = true
+		}
+	}
+}
+
+func sayError(text string) {
+	sayWithPrefix(text, "ERROR ")
+}
+
+func sayWarning(text string) {
+	sayWithPrefix(text, "⚠ ")
+}
+
+func sayInfo(text string) {
+	sayWithPrefix(text, "> ")
+}
+
+func sayInfoIndented(text string) {
+	sayWithPrefix(text, "    ")
+}
+
+func sayIndented(text string) {
+	sayWithPrefix(text, "  ")
+}
+
+func sayFix(instruction string, command string) {
+	sayWithPrefix(instruction, "👉 ")
+	sayEmptyLine()
+	sayIndented(command)
+	sayEmptyLine()
+}
+
+func sayNext(instruction string, command string) {
+	sayWithPrefix(instruction, "👉 ")
+	sayEmptyLine()
+	sayIndented(command)
+	sayEmptyLine()
+}
+
+func sayWithPrefix(s string, prefix string) {
+	lines := strings.Split(strings.TrimSpace(s), "\n")
+	for i := 0; i < len(lines); i++ {
+		printToConsole(prefix + strings.TrimSpace(lines[i]) + "\n")
+	}
+}
+
+func sayEmptyLine() {
+	printToConsole("\n")
+}
+
+func say(s string) {
+	if len(s) == 0 {
+		return
+	}
+	printToConsole(strings.TrimRight(s, " \r\n\t\v\f") + "\n")
+}
+
+func debugInfo(text string) {
+	if Debug {
+		sayWithPrefix(text, "DEBUG ")
+	}
+}
+
+var printToConsole = func(message string) {
```

Comment:
why is this an anonymous function?

Reply:
tried making it a func. found out that its overridden in mob_test so leaving it for now.